content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx

```diff
@@ -24,17 +24,17 @@ generator client {
 }
 ```
 
-## Create and modify queries
+## Extend Prisma Client query operations
 
 Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an [extended client](/concepts/components/prisma-client/client-extensions#about-prisma-client-extensions). An extended client is a variant of the standard Prisma Client that is wrapped by one or more extensions.
 
-Use the `query` extension component to create and modify queries. You can add or modify a custom query in the following:
+Use the `query` extension component to modify queries. You can modify a custom query in the following:
 
-- [A specific operation in a specific model](#add-or-modify-a-custom-query-for-a-specific-operation-in-a-specific-model)
-- [A specific operation in all models of your schema](#add-or-modify-a-custom-query-for-a-specific-operation-in-all-models-of-your-schema)
-- [All operations in a specific model](#add-or-modify-a-custom-query-for-all-operations-in-a-specific-model)
-- [All operations in all models of your schema](#add-or-modify-a-custom-query-for-all-operations-in-all-models-of-your-schema)
-- [A specific top-level client operation](#add-or-modify-a-custom-query-for-a-specific-top-level-client-operation)
+- [A specific operation in a specific model](#modify-a-specific-operation-in-a-specific-model)
+- [A specific operation in all models of your schema](#modify-a-specific-operation-in-all-models-of-your-schema)
+- [All operations in a specific model](#modify-all-operations-in-a-specific-model)
+- [All operations in all models of your schema](#modify-all-operations-in-all-models-of-your-schema)
```

Comment:
I think these 4 it is actually about modifying or adding though?

Reply:
This section only describes modifying a query.

There is another section that describes how to add methods here:

https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions/model

The way you add functions is a little bit different than extending.

Reply:
Yes, `query` technically modifies method behavior and `model` adds new methods (or can even override existing ones).

Reply:
Ah cool. Cross links might be nice.